app/uk/gov/hmrc/helptosavefrontend/views/helpinformation/how_sign_in.scala.html

```diff
@@ -0,0 +1,20 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@()(implicit messages: play.api.i18n.Messages)
+
+<p class="bold">@Messages("hts.help-information.section.how-account-works.s2.p1")</p>
+<p>@Html(Messages("hts.help-information.section.how-account-works.s2.p2", "http://www.gov.uk/hmrc/app"))</p>
```

Comment:
this URL seems to belong with the other URL's in application.conf (https://github.com/hmrc/help-to-save-frontend/blob/master/conf/application.conf#L225). If the URL needs changing in the future it'd be easier if it was read from application.conf. The same principle applies to the youtube embed codes - it'd be easier change both of them if we need to in the future if they sat in application.conf

Reply:
Have added these URLs to the app config.